### PR TITLE
 Add installation logic to workflow. Similar to perturbation install logic

### PR DESCRIPTION
**Pull request by jpa198**
Add required qpqs parameters to emod3d_defaults. These will be ignored if not used
Remove unused emod3d_defaults parameters

#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Comment by jpa198** [comment]
Jenkins failing on changes to e3d.par and qcore dependency

File: scripts/cybershake/install_cybershake.py [hunk 1]
```diff
@@ -74,81 +72,93 @@ def main():
...
         help="Don't use velocity model perturbations. If this is selected any perturbation files will be ignored.",
     )
-
+    parser.add_argument(
+        "--vm_qpqs_files",
+        action="store_true",
+        help="Use generated Qp/Qs files. If this is selected all events/faults must have Qp and Qs files.",
+    )
+    parser.add_argument(
+        "--ignore_vm_qpqs_files",
```
Comment (jasonmotha) [inline scripts/cybershake/install_cybershake.py:81]: You can make these two arguments exclusive.

You could even store it in one variable. A nullable bool. True - use it, False ignore it and None - don't expect it

Comment (jasonmotha) [reply scripts/cybershake/install_cybershake.py:81]: https://riptutorial.com/python/example/6133/setting-mutually-exclusive-arguments-with-argparse

**Review by claudio525 — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

File: templates/gmsim/20.4.1.4/emod3d_defaults.yaml [hunk 2]
```diff
@@ -57,7 +59,5 @@ ts_xy: 1
 ts_xz: 0
 ts_yz: 0
 tzero: 0.6
+use_qpqs: 0
 vmodel_swapb: 0
-xseis: 0
-yseis: 0
-zseis: 0
```
Comment (shot0829) [inline templates/gmsim/20.4.1.4/emod3d_defaults.yaml:63]: why was these originally here and now we are removing them from all gmsim template?

**Review by shot0829 — APPROVED** [review]

**Pull request merged**